riemann/tx/tx.py

```diff
@@ -191,12 +193,14 @@ def from_bytes(VarInt, byte_string):
             raise ValueError('Malformed VarInt. Got: {}'
                              .format(byte_string.hex()))
 
-        ret = VarInt(utils.le2i(num))
-
-        if non_compact:
+        if non_compact and 'overwinter' in riemann.get_current_network_name():
             raise ValueError('VarInt must be compact. Got: {}'
                              .format(byte_string.hex()))
 
+        ret = VarInt(
```

Comment:
Did you intended to use `ret` again in this method? Return statement directly if not.